### DISCUSSION TITLE
Dialog to authorize Copilot in Visual Studio 2022 not showing

### DISCUSSION THREAD

**Discussion by charming-byte** [community/community#33528] [Copilot Conversations]
I just installed Github Copilot for Visual Studio 2022. 
I'd expect that I get prompted after restarting my IDE to authorize the extension as described [here](https://github.blog/2022-03-29-github-copilot-now-available-for-visual-studio-2022)  but unfortunately nothing happens. I tried to reinstall copilot just to be sure, but that didnt had any effect.

The output of Github Copilot extension is

>GitHub Copilot 1.44.0.0 (v1.44.0.0@47e914df6)
> GitHub Copilot Agent v1.44.0

Does anyone know how to fix this?

**Comment by iamcjwilliams** [comment]
@thiago-dev can you provide more detail? The image link provided does not display a reference to copilot
